### PR TITLE
Fix Hungarian default date format and add Hungarian translations

### PR DESCRIPTION
**Pull request by tamasori**
Hey,

I fixed the Hungarian default date format as it was incorrect, you can find more information here:
https://en.wikipedia.org/wiki/Date_and_time_notation_in_Hungary#Date

Also I added translations for the Hungarian language!

### CODE REVIEW

File: src/Countries/Support/Translations/hu.php [hunk 1]
```diff
@@ -0,0 +1,285 @@
+<?php
+
+namespace Macmotp\Countries\Support\Translations;
+
+use Macmotp\Continent;
+use Macmotp\Country;
+
+/**
+ * English
```
Comment (mihaituhari) [inline src/Countries/Support/Translations/hu.php:9]: **issue:** 
```suggestion
 * Hungarian
```

**Review by mihaituhari — CHANGES_REQUESTED** [review]
👋  I noticed a typo

**Comment by macmotp** [comment]
Thank you @tamasori, since I am going to release a more stable version soon, I will keep this PR on hold and apply your changes directly. I will add you as contributor for reference

**Comment by tamasori** [comment]
Ok, thank you @macmotp

**Comment by macmotp** [comment]
All your changes have been included into the new release, thanks!

**Pull request closed without merging**